docs/deprecations.md

```diff
@@ -363,3 +363,12 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 Deprecation code: `BIN_SERVERLESS`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
+
+<a name="CLI_DEPLOY_FUNCTION_OPTION"><div>&nbsp;</div></a>
+
+## CLI deploy command function option removed
```

Comment:
Let's phrase the whole message so it explains that the flag has been deprecated and will be removed with next major (`3.0.0`).